### PR TITLE
update message to work around HTTP method validation

### PR DESCRIPTION
**Pull request by danibusu**
Update the sent message to start with a letter other than ones of valid
HTTP methods to bypass gateway validation. In a real scenario, the
request would time-out.

### CODE REVIEW

**Comment by danibusu** [comment]
@jitsni This is the PR for the HTTP method validation issue.

**Comment by sbadugu** [comment]
@jitsni Can you take an action item on this PR ?

File: specification/httpx/src/main/scripts/org/kaazing/specification/httpx/extended/client.sends.message.between.opening.and.extended.handshake/request.rpt [hunk 1]
```diff
@@ -38,21 +38,12 @@ read header "Date" /.+/
 
 write [0x82 0x85] ${writeMask}
 write option mask ${writeMask}
-write "Hello"
+write "World"
```
Comment (jitsni) [inline specification/httpx/src/main/scripts/org/kaazing/specification/httpx/extended/client.sends.message.between.opening.and.extended.handshake/request.rpt]: I see what happens in HTTP decoding when Hello is sent as method. Can you not just send "Hello World"

**Comment by cmebarrow** [comment]
@danibusu Does this still need to get merged. If so, please answer Jitu's review comment.

**Comment by dpwspoon** [comment]
not needed

**Pull request closed without merging**